front-end/src/components/modal/create/AnalyzersAndSeries/AnalyzersAndSeries.jsx

```diff
@@ -84,7 +84,7 @@ const AnalyzersAndSeries = ({
     const newSeries = {
       'columns': selectColumns,
       'analyze': {
-        'analysis': checkedAnalyzer.code,
+        'type': checkedAnalyzer.code,
```

Comment:
Why did you rename this to type?

Reply:
Analysis didn't really make sense, backend calls it 'type' so I renamed.